cmake/FindEOSPAC.cmake

```diff
@@ -24,7 +24,7 @@
 FIND_PATH(EOSPAC_INCLUDE_DIR NAMES eos_Interface.h)
 
 # Look for the library.
-FIND_LIBRARY(EOSPAC_LIBRARY NAMES eospac6 libeospac6)
+FIND_LIBRARY(EOSPAC_LIBRARY NAMES eospac6 libeospac6 libeospac6gpu.a)
```

Comment:
what happens if hte GPU library can't be found?

Reply:
it will attempt to find a library that matches any in that list, so it should be fine. I'll test to make sure.

Reply:
I confirmed this is the case @Yurlungur